bitnami-postgis/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG PG_VERSION_TAG
+FROM timescale/timescaledb:1.4.2-${PG_VERSION_TAG}-bitnami
```

Comment:
Perhaps we could provide `--build-arg VERSION=$(shell awk '/^ENV TIMESCALEDB_VERSION/ {print $$3}' Dockerfile)`